pkg/controller/statefulset/stateful_controller_test.go

```diff
@@ -0,0 +1,361 @@
+package statefulset
```

Comment:
This file is misnamed, should be `statefulset_controller_test.go`.

Could you also please add the license header to this file.